src/sync/app.cpp

```diff
@@ -628,6 +628,9 @@ void App::get_profile(std::shared_ptr<SyncUser> sync_user,
 
             sync_user->set_state(SyncUser::State::LoggedIn);
             m_sync_manager->set_current_user(sync_user->identity());
+            for (auto& [_, subscriber] : m_subscribers) {
```

Comment:
Is there any way to add a facade over all of these `for`'s so it would become `emit_change_to_subscribers(*this)`?

Reply:
Yeah, would clean things up a bit.